src/sage/matrix/matrix_double_dense.pyx

```diff
@@ -1739,14 +1739,13 @@ cdef class Matrix_double_dense(Matrix_numpy_dense):
             import scipy
         import scipy.linalg
         X = self._new(self._ncols, B.ncols())
-        arr, resid, rank, s = scipy.linalg.lstsq(self._matrix_numpy, B.numpy())
+        arr, _, _, _ = scipy.linalg.lstsq(self._matrix_numpy, B.numpy())
```

Comment:
do you know this funny Python syntax ? could help avoiding `_, _, _`
```
sage: a, *b = list(range(5))
sage: a, b
(0, [1, 2, 3, 4])
sage: *a, b = list(range(5))
sage: a, b
([0, 1, 2, 3], 4)
sage: a, *_, b = list(range(50))
sage: a, b
(0, 49)
```

Reply:
merci. J'ai choisi de faire un truc plus simple.